### ISSUE TITLE
fast sensor dynamics in Storage.Stratified

### ISSUE THREAD

**Issue by Mathadon** [lbl-srg/modelica-buildings#349]
```Buildings.Fluid.Storage.Stratified``` contains enthalpy sensors ```hA_flow```, ```hB_flow``` and ```hVol_flow```. These sensors have very fast dynamics (tau=1), which are not propagated. Furthermore these sensors are not even used by the model (they are however used in ```Buildings.Fluid.Storage.StratifiedEnhanced```). The fast dynamics of these sensors create a bottleneck for the simulation speed. By disabling the dynamics (tau = 0) I was able to reduce the required number of simulation steps by 2.5% in one of my models (100k equations).

I propose to set tau = 0 by default in ```Buildings.Fluid.Storage.Stratified``` and to propagate its value in ```Buildings.Fluid.Storage.StratifiedEnhanced``` with a default value larger than 1 second. I will propose an implementation.

**Comment by rubenbaetens** [comment]
Furthermore, i propose to make them conditional ? If you say they're not even used.

**Comment by mwetter** [comment]
Thanks for the changes. I merged your pull request #350 to a new branch `issue349_storageDynamics` and further edited the model.
I did the following (besides some code clean-up):

1. Added <code>final</code> to <code>tau = 0</code> in <code>EnthalpyFlowRate</code>. These sensors do not need dynamics as the enthalpy flow rate is used to compute a heat flow which is then added to the volume of the tank. Thus, if there were high frequency oscillations of small mass flow
rates, then they have a small effect on <code>H_flow</code>, and they are not used in any control loop. Rather, the oscillations are further damped by the differential equation of the fluid volume. Hence, these <code>tau</code> are now zero for all tank models.
2. I improved the documentation of <code>tau</code>, which is used in the <code>buo</code> instance.  (It was a bad implementation to use the same variable name for two very different things, but this is now fixed.)

Let me know if you see any issues with the above. I will run some more tests and then merge to the master.
